python/lsst/ts/xml/data/sal_interfaces/MTMount/MTMount_Commands.xml

```diff
@@ -10,6 +10,13 @@
     <Subsystem>MTMount</Subsystem>
     <EFDB_Topic>MTMount_command_openMirrorCovers</EFDB_Topic>
     <Description>Open the mirror covers to allow light in. Warning: This terminology may be the opposite of that used by the Engineering User Interface.</Description>
+    <item>
+      <EFDB_Name>leaf</EFDB_Name>
+      <Description>Specify a leaf to open. By default (leaf=0) open all.</Description>
```

Comment:
What other values are possible? I assume it is a single value for a specific leaf. What are the valid values and to which leafs do they correspond? Please introduce an enum for all values and to which leaf they correspond and indicate here that that enum should be used.